app/models/zip_code.rb

```diff
@@ -35,7 +35,7 @@ def single_district?
   end
 
   def single_district
-    single_district? && districts.first
+    districts.first if single_district?
   end
```

Comment:
I guess I went overboard changing `a if a` to `a && a` -- in this case I want it to return `nil`, not `false`